libs/components/modals/testing/src/confirm/confirm-button-harness.ts

```diff
@@ -0,0 +1,56 @@
+import { ComponentHarness, HarnessPredicate } from '@angular/cdk/testing';
+
+import { SkyConfirmButtonHarnessFilters } from './confirm-button-harness-filters';
+
+type ButtonStyle = 'primary' | 'default' | 'link';
```

Comment:
If we're returning this type from a public method, it needs to be prefixed with `Sky`. Also, we should probably move this to the source code directory and update the `SkyConfirmButton` interface to be:
```
// TODO: Remove 'string' in a breaking change.
styleType: SkyConfirmButtonStyleType | string;
```